.travis.yml

```diff
@@ -33,8 +33,8 @@ install:
 before_script:
   - set -o errexit; source .travis/test_05_before_script.sh
 script:
-  - if [ $SECONDS -gt 1200 ]; then set +o errexit; echo "Travis early exit to cache current state"; false; else set -o errexit; source .travis/test_06_script_a.sh; fi
-  - if [ $SECONDS -gt 1800 ]; then set +o errexit; echo "Travis early exit to cache current state"; false; else set -o errexit; source .travis/test_06_script_b.sh; fi
+  - if [ $SECONDS -gt 1200 ]; then set +o errexit; echo "Early exit to cache current state. Please re-run job"; false; else set -o errexit; source .travis/test_06_script_a.sh; fi
```

Comment:
Thanks, that's really helpful. Would suggest prefixing the error message with `Error:` and also explaining more:

```sh
echo "Error: Initial build successful, but not enough time remains to run tests. Please manually re-run job by using the travis restart button or asking a bitcoin maintainer to restart. The next run should not time out, because ccache build output was saved."
```